lib/src/user_tag.dart

```diff
@@ -0,0 +1,65 @@
+import 'package:linkify/linkify.dart';
+
+final _userTagRegex = RegExp(
+  r'^(.*?)(?<![\w@])@([\w@]+(?:[.!][\w@]+)*)',
```

Comment:
(I know other linkers don't have this, but...)

Could you add a regex101.com (or similar) link (as a comment) with what this regex matches (and doesn't match)?